content/chat/setup.textile

```diff
@@ -138,13 +158,27 @@ const App = () => {
 };
 ```
 
+```[swift]
+let realtimeOptions = ARTClientOptions()
+realtimeOptions.key = "<API_KEY>"
```

Comment:
For consistency, would it be worth having their API_KEY automatically used as we have with the JS and React examples?

```suggestion
realtimeOptions.key = "{{API_KEY}}"
```

Reply:
The same for any reference of code that uses a key with Swift examples?

Reply:
Good shout, fixed!